beacon-chain/rpc/eth/beacon/blinded_blocks.go

```diff
@@ -202,31 +203,72 @@ func (bs *Server) SubmitBlindedBlockSSZ(ctx context.Context, req *ethpbv2.SSZCon
 		return &emptypb.Empty{}, status.Errorf(codes.Internal, "Could not unmarshal request data into block: %v", err)
 	}
 
-	if block.IsBlinded() {
+	switch forkVer {
+	case bytesutil.ToBytes4(params.BeaconConfig().CapellaForkVersion):
+		if !block.IsBlinded() {
+			return nil, status.Error(codes.InvalidArgument, "Submitted block is not blinded")
+		}
+		b, err := block.PbBlindedCapellaBlock()
+		if err != nil {
+			return &emptypb.Empty{}, status.Errorf(codes.Internal, "Could not get proto block: %v", err)
+		}
+		_, err = bs.V1Alpha1ValidatorServer.ProposeBeaconBlock(ctx, &eth.GenericSignedBeaconBlock{
```

Comment:
When processing SSZ, I avoid calling the helper `submitXXX` functions because I already have a v1alpha1 block, and they require a v1/v2 block that they convert to v1alpha1. So that means two unnecessary conversions.